job/models.py

```diff
@@ -43,8 +43,9 @@
 
 log = logging.getLogger(__name__)
 """
-
-            
+When editing this file, help_text ,ust have the following punctuation:
```

Comment:
@Ramdan42 you have a typo on this line.

Also, it would be nice to have examples of the 2 conditions for clarity.